packages/website/bin/page-generator/get-docs-info.js

```diff
@@ -2,7 +2,7 @@
 // Does not build pages at this point.
 
 const path = require('path');
-const fs = require('fs');
+const fs = require('fs-extra');
```

Comment:
So should we just never use the inbuilt fs package?

Reply:
Preferably, just so all behaviour is consistent.